boot/freeldr/freeldr/ntldr/arch/i386/winldr.c

```diff
@@ -178,6 +174,8 @@ ULONG PcrBasePage;
 ULONG TssBasePage;
 PVOID GdtIdt;
 
+extern BOOLEAN PaeModeOn;
```

Comment:
Could be worth placing in `boot/freeldr/freeldr/ntldr/winldr.h` at line 76 (below the `// winldr.c`)

Reply:
Moved